packages/docusaurus-utils/src/urlUtils.ts

```diff
@@ -83,6 +83,13 @@ export function normalizeUrl(rawUrls: string[]): string {
   return str;
 }
 
+/**
+ * Some slugs that seem valid may be dynamic routes for React router. We sanitize these paths.
+ */
+export function sanitizeURL(url: string): string {
+  return url.replace(/(?:(?<=\/):)|[()*[\]{}]/g, '');
```

Comment:
Rather than replace these with empty string, would it be preferable to replace with some safe character (e.g. `-` or `_`)? Seems like that would reduce the chance of collision somewhat.

Reply:
Yes, I'd be curious as what other site generators are doing—we obviously want to align with them if they do something sane in this case!